python/test/CMakeLists.txt

```diff
@@ -237,6 +237,7 @@ ot_pyinstallcheck_test (NonStationaryCovarianceModelFactory_std)
 ot_pyinstallcheck_test (ProductCovarianceModel_std)
 ot_pyinstallcheck_test (StationaryFunctionalCovarianceModel_std IGNOREOUT)
 ot_pyinstallcheck_test (LatentVariableModel_std IGNOREOUT)
+ot_pyinstallcheck_test (ProcessAndFieldOperators_std)
```

Comment:
we should prefer assert-based tests over expout files
 (its ok to test only one component to simplify testing as much as possible)

Reply:
This choice was due to the impossibility of asserting equality between objects other than samples & points, and I chose  a more thorough testing over non-unitarty size & multi-dimensional fields.
I am open to simplify the tests to assert based ones if necessary

Reply:
Perhaps you could introduce a new assert_almost_equal_field() function? It would check that a computed Field is almost equal to a reference Field. This would be the building block of your test (and perhaps other unit tests as well). You would need to implement a new assert_almost_equal_mesh(). These codes would locate into ottest. They could be based on assert on Points and Sample, which already exist.

Reply:
I added assert operators for meshes, fields and processsamples for future testing

Reply:
thanks a lot